CHANGELOG.md

```diff
@@ -2,10 +2,17 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
```

Comment:
You also need to update the ThirdPartyNotices file. Ping me if you need help with that